models/engine/db_storage.py

```diff
@@ -74,3 +74,25 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Returns the object based on the class name and its ID """
+        cls_dict = {}
+        cls_dict = self.all(cls)
+
+        if cls is not None:
```

Comment:
I liked this conditional, so you don't add any unnecessary load to the database in case no class is given.